F2P-139 | Potential issue with `updateWebsiteUserEmailAddress`

aldrichdev opened the pull request:
# What's Changed

- Updated email change logic to send email to old address when website email is changed

# Testing Notes

Confirmed an email is sent to the old email address when a website user's email is changed, and the template variable works (and shows the new address).

BeastFable commented on src/pages/account/change-email/[id].tsx:
```diff
@@ -25,12 +26,20 @@ const ChangeEmailByIdPage = () => {
+            recipient: user.emailAddress,
+            newEmail,
+          })
+          .then(() => {
```
I would add a `.catch` after this that logs any error with EmailJS sending the email, and stops loading... since, if the EmailJS functionality breaks, it doesn't mean the main operation failed, so it should stop loading and show the success message even if it does.

aldrichdev replied:
Done.

aldrichdev commented:
I think this ticket needs more iterations. I have noticed a few issues.
1. `updateWebsiteUserEmailAddress` runs multiple times in `change-email/[id]`. I have fixed this in a local stash.
2. I added code to log the user out and then back in when the notice email is sent to the old address; however, this doesn't work consistently. Sometimes we get `403` Forbidden on the next email update. Other times I get Access Denied when navigating to the account page after the update is successful.
3. The notice email to the old address is never sent to `charadicks@gmail.com` every time I've changed my email from that to another one.

BeastFable reviewed (approved):
LGTM
